docs/en/observability/synthetics-scaling-and-architecting.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[synthetics-scaling-and-architecting]]
+= Scaling and Architecting a Synthetics Deployment
+
+While for simple use cases little to no planning needs to be taken in the implementation of *{synthetics-app}*, 
+for complex use cases and at a high scale this section describes advanced considerations for any *{synthetics-app}*
+implementation
```

Comment:
I think this can be simplified. I'm not sure this is perfect, but maybe something like: 

```suggestion
Use these advanced considerations when using the {synthetics-app}
for large and complex use cases.
```